firebase-database/src/androidTest/java/com/google/firebase/database/QueryTest.java

```diff
@@ -1427,6 +1784,25 @@ public void nullPrioritiesIncludedInStartAt2() throws DatabaseException, Interru
     DeepEquals.assertEquals(expected, result);
   }
 
+  @Test
+  public void nullPrioritiesIncludedInStartAfter2() throws DatabaseException, InterruptedException {
```

Comment:
Do you know what the `2` is meant to indicate? If not, I suggest we remove it from here and above.